src/wavesurfer.ts

```diff
@@ -130,6 +130,8 @@ export type WaveSurferEvents = {
   zoom: [minPxPerSec: number]
   /** Just before the waveform is destroyed so you can clean up your events */
   destroy: []
+  /** When browser is unable to fetch media file */
```

Comment:
What about media errors and decoding errors?

Reply:
```suggestion
  /** When media files is unable to be fetched, decoded, or an error is thrown by media element */
```

does this look better?

Reply:
Yes, perfect!